assets/models/system/jean-baptiste-camembert-ner/spec.yaml

```diff
@@ -1,5 +1,5 @@
 $schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
-name: Jean-Baptiste-camembert-ner
+name: jean-baptiste-camembert-ner
```

Comment:
I assume the existing `Jean-Baptiste-camembert-ner` will need to be archived as well?

Reply:
I think yes. @skasturi Could you please confirm as well?